src/Ledger/GovernanceActions.lagda

```diff
@@ -112,7 +183,48 @@ NeedsHash (TriggerHF _)        = GovActionID
 NeedsHash (ChangePParams _ _)  = GovActionID
 NeedsHash (TreasuryWdrl _)     = ⊤
 NeedsHash Info                 = ⊤
+\end{code}
 
+A governance action is an on-chain event that is triggered by a transaction, and there is a deadline after which
+the governance action cannot be enacted.
+\begin{itemize}
+\item
+  \defn{Ratification}. An action is said to be \defn{ratified} when it gathers enough votes in its favor (through the rules and parameters that are detailed
```

Comment:
These can be more specific, we can point to the sections/figures that contain the actual logic.

Reply:
I added a reference to the separate section/module on Ratification (\ref{sec:ratification}).